docs/UserGuide.md

```diff
@@ -98,6 +98,7 @@ Shows a message listing out all the available commands and their purpose.
 Format: `help`
 
 For more information regarding the command formats and examples, press `F1` to open up a help window (as shown in the picture below).
+  * For some users with newer Macs, use `fn`+`F1` to open the help window.
```

Comment:
This seems oddly specific (some laptop keyboard also has this Fn lock). Just say "Depends on your keyboard, you might have to press other keyboard combinations, such as Fn+F1, to trigger F1".

For all you know I'm pressing `SPAOFRLG` to open the help window.

Reply:
Yeah but it's specific to the newer Macs right

Reply:
My laptop has Fn lock key too. Something like this https://en.wikipedia.org/wiki/F-Lock